### ISSUE TITLE
Error when creating slash command: Application command interaction option values must be of type string, integer, or boolean

### ISSUE THREAD

**Issue by Quadrubo** [interactions-py/interactions.py#238]
### Description
Slash command fails every second time for no apparent reason
### Steps to reproduce
Create the following subcommand:
```
@cog_ext.cog_subcommand(base="test",
                            subcommand_group="add",
                            name="user",
                            options=[
                                create_option(
                                    name="user",
                                    description="Description 1",
                                    option_type=6,
                                    required=True
                                ),
                                create_option(
                                    name="date",
                                    description="Description 2.",
                                    option_type=3,
                                    required=True
                                ),
                                create_option(
                                    name="name",
                                    description="Description 3.",
                                    option_type=3,
                                    required=False
                                )
                            ])
```

**Comment by LordOfPolls** [comment]
No you don't. The library converts the classes you're passing into an int that discord understands. The numbers OP is using are what the option types are actually represented as

**Comment by Toricane** [comment]
Worked for me all the times
cog.py:
```py
from discord.ext import commands
from discord_slash import cog_ext
from discord_slash.utils.manage_commands import create_option

class cog(commands.Cog):
    def __init__(self, bot):
        self.bot = bot
    
    @cog_ext.cog_subcommand(base="test",
                                subcommand_group="add",
                                name="user",
                                options=[
                                    create_option(
                                        name="user",
                                        description="Description 1",
                                        option_type=6,
                                        required=True
                                    ),
                                    create_option(
                                        name="date",
                                        description="Description 2.",
                                        option_type=3,
                                        required=True
                                    ),
                                    create_option(
                                        name="name",
                                        description="Description 3.",
                                        option_type=3,
                                        required=False
                                    )
                                ])
    async def test_user_add_user(self, ctx, user, date, name=None):
        await ctx.send("Works")

def setup(bot):
    bot.add_cog(cog(bot))
```
Tested global and guild_id, worked

**Comment by Toricane** [comment]
Any error that you have?

**Comment by Quadrubo** [comment]
> Any error that you have?

The Error only Exists in the Discord Client in DMs, try to reproduce it there

**Comment by Toricane** [comment]
Is the error:
`Application command interaction option values must be of type string, integer, or boolean`
Hmm yeah I got it exactly every other time, I don't know why

**Comment by Toricane** [comment]
> Is the error:
> `Application command interaction option values must be of type string, integer, or boolean`
> Hmm yeah I got it exactly every other time, I don't know why

Oh wait I'm dumb, didn't see that in the title

**Comment by Quadrubo** [comment]
Can anyone test this on discord.js or something else? I also get this error on servers now with a different command, If someone could test it there we'd know if its a discord error or not.

**Comment by i0bs** [comment]
This seems to be replicable on my end and has some involvement with how Discord handles option values for DM channels, marking as a bug for now.

**Comment by i0bs** [comment]
Bug will be resolved with v4.0
